SachinSinghDighan/RecipeRoulette/app/src/main/java/com/example/reciperoulette/presentation/recipedetail/RecipeDetailActivity.kt

```diff
@@ -0,0 +1,317 @@
+package com.example.reciperoulette.presentation.recipedetail
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.animation.animateContentSize
+import androidx.compose.animation.core.Spring
+import androidx.compose.animation.core.spring
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.paddingFromBaseline
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyListScope
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ExpandLess
+import androidx.compose.material.icons.filled.ExpandMore
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.make_food.ui.commonui.AppBar
+import com.example.make_food.ui.commonui.HorizontalListImage
+import com.example.make_food.ui.commonui.HorizontalListTitleText
+import com.example.make_food.ui.commonui.RecipeImage
+import com.example.make_food.ui.commonui.ScreenContentText
+import com.example.make_food.ui.commonui.ScreenSubTitleText
+import com.example.make_food.ui.commonui.ScreenTitleText
+import com.example.reciperoulette.R
+import com.example.reciperoulette.presentation.recipedetail.ui.theme.RecipeRouletteTheme
+
+class RecipeDetailActivity : ComponentActivity() {
```

Comment:
why we need 2 activity?

Reply:
Removed 2nd activity and created different composed functions for different screens